utils/test.js

```diff
@@ -6,6 +6,10 @@ const {setPassword, sleep, initServer, startServer, spawnBots, helpers, logConso
 
 const {cliPort, verbose, tickDuration, playerRoom, players, rooms, milestones} = require('./testConfig');
 
+// todo for local-testing
+// if your machine is slow try increment this
+const waitForConnection = 10;
```

Comment:
I would extract this into the `testConfig.js` as there are all the other configuration values aswell

Reply:
done